### PR TITLE
Completed es6-const-let. | Vera Yeh

### PR DESCRIPTION
**Pull request by VeraYeh**
image

### CODE REVIEW

File: es6-const-let/index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      let i = 0;
+      while (i < greetingList.length) {
+        console.log('Current greeting:', greetingList[i]);
+        i++;
+      }
+    }
+    logAllGreetings(greetings);
+
+    function getRandomGreeting(greetingList) {
+      let randomIndex = Math.floor(Math.random() * greetingList.length);
```
Comment (BrettAlbright) [inline es6-const-let/index.html]: This one should be a `const`

File: es6-const-let/index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    }
+    console.log('Random greeting:', getRandomGreeting(greetings));
+
+    function addToGreetings(newGreeting, greetingList) {
+      greetingList.push(newGreeting);
+    }
+    addToGreetings('namaste', greetings);
+    logAllGreetings(greetings);
+
+    let variable = 'meow';
```
Comment (BrettAlbright) [inline es6-const-let/index.html]: This one should also be a `const` The reason for this one and the above one being consts is that their values are never reassigned.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
a couple small updates!

File: es6-const-let/index.html (hunk 2 above)
Comment (VeraYeh) [reply es6-const-let/index.html]: Fixed above two variables accordingly. Thanks.
image

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**